uelc/templates/500.html

```diff
@@ -1,4 +1,4 @@
-
+{% extends "base.html" %}
```

Comment:
extending the base template on the 500 page will break because of compression issues. This needs to be removed.